### PR TITLE
readme improvements

### PR DESCRIPTION
**Pull request by aniham**

### CODE REVIEW

File: examples/events/example-contexts/mock-search-results-context.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 [Search Results schema](../../../packages/storefront-events-sdk/src/types/schemas/searchResults.ts)
 
-_Note_ It is important that `searchUnitId` matches `searchUnitId` in request for the context to be processed by Commerce correctly.
+**IMPORTANT** `searchUnitId` must match `searchUnitId` in request for the context to be processed by Commerce correctly.
```
Comment (dobooth) [inline examples/events/example-contexts/mock-search-results-context.md:15]: Should we clarify that `x.searchUnitId` must match `y.searchUnitId`?

File: packages/storefront-events-sdk/README.md [hunk 2]
```diff
@@ -93,531 +93,35 @@ Addiontally, you can publish custom events or add a custom context to a supporte
...
-mse.context.getSearchResults();
-mse.context.getShopper();
-mse.context.getShoppingCart();
-```
+Required contexts per event under [examples/events/](../../examples/events/).
 
 ### Publish
 
-These functions publish events which notify all subscribers:
+Pre-defined publishable events listed in the [PublishManager](./src/PublishManager.ts). Any event that's not part of the `PublishManager` should be handled as a custom event.
```
Comment (dobooth) [inline packages/storefront-events-sdk/README.md:108]:
```suggestion
Pre-defined publishable events are listed in the [PublishManager](./src/PublishManager.ts). Any event that is not part of the `PublishManager` should be handled as a custom event.
```

File: packages/storefront-events-sdk/README.md [hunk 3]
```diff
@@ -93,531 +93,35 @@ Addiontally, you can publish custom events or add a custom context to a supporte
...
-```
-
-```javascript
-mse.publish.signOut(ctx);
-```
-
-```javascript
-mse.publish.updateCart(ctx);
-```
+A list of examples on publishing each event is available under [examples](../../examples/).
```
Comment (dobooth) [inline packages/storefront-events-sdk/README.md:110]:
```suggestion
A list of examples on publishing events is available under [examples](../../examples/).
```

File: packages/storefront-events-sdk/README.md [hunk 4]
```diff
@@ -93,531 +93,35 @@ Addiontally, you can publish custom events or add a custom context to a supporte
...
 
 ### Subscribe
 
-These functions subscribe to events:
+Supported subscibe events are defined in the [SubscribeManager](./src/SubscribeManager.ts). Any event that's not part of the `SubscribeManager` should be handled as a custom event.
```
Comment (dobooth) [inline packages/storefront-events-sdk/README.md:114]:
```suggestion
Supported `subscibe` events are defined in the [SubscribeManager](./src/SubscribeManager.ts). Any event that is not part of the `SubscribeManager` should be handled as a custom event.
```

File: packages/storefront-events-sdk/README.md [hunk 5]
```diff
@@ -93,531 +93,35 @@ Addiontally, you can publish custom events or add a custom context to a supporte
...
-mse.subscribe.updateCart(handler, options);
-```
+Note that a large number of custom events with different schemas is an anti-pattern (most events you need should be supported by `SubscribeManager`). If there are standard events you believe should be supported by `SubscribeManager`, please submit an [issue](https://github.com/adobe/commerce-events/issues).
+
+Note that it is rare for an application utilizing the SDK to have to subscribe to events. In most cases subscribes and unsubscribes are handled by the [Storefront Events Collector](../storefront-events-collector/).
 
 ### Unsubscribe
 
-These functions unsubscribe from events:
+Supported unsubscibe events are defined in the [UnubscribeManager](./src/UnsubscribeManager.ts).
```
Comment (dobooth) [inline packages/storefront-events-sdk/README.md:122]:
```suggestion
Supported `unsubscibe` events are defined in the [UnubscribeManager](./src/UnsubscribeManager.ts).
```

**Review by grahamcrackers — APPROVED** [review]

**Pull request merged**